docs/docs/docs/01-core/admin/04-hooks/use-admin-users.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: useAdminUsers
+description: API reference for the useAdminUsers hook
+tags:
+  - admin
+  - hooks
+  - users
+---
+
+An abstraction around `react-query`'s `useQuery` hook. It can be used to fetch one ore more admin users.
+
+## Usage
+
+The hooks can receive two optional parameters:
+
+1. query params: an object containing the query params to be sent to the API. They are going to be
+   stringified by `qs`. All params are equal except `id`, which is used to fetch a single users, if
+   it is passed.
+2. options: an object containing the options to be passed to `useQuery`.
+
+It returns an object containing some of the react-query attributes.
+
+## Typescript
+
+```ts
+import { UseQueryOptions } from 'react-query'
+
+type User = object;
```

Comment:
I would rather not, because I'd have to type the admin API then, which is a different task :D I think for the user of this docs it is enough to understand what shape it should have.

Reply:
Sounds good 👍